parser/src/test/scala/org/polystat/odin/parser/xmir/ParserTests.scala

```diff
@@ -0,0 +1,131 @@
+package org.polystat.odin.parser.xmir
+
+import cats.effect.testing.scalatest.AsyncIOSpec
+import cats.effect.{IO, Sync}
+import cats.implicits._
+import fastparse.Parsed
+import org.polystat.odin.core.ast.EOBnd
+import org.polystat.odin.core.ast.astparams.EOExprOnly
+import org.polystat.odin.parser.MutualRecExample
+import org.polystat.odin.parser.fastparse.Parser
+import org.scalatest.Assertion
+import org.scalatest.freespec.AsyncFreeSpec
+
+class ParserTests extends AsyncFreeSpec with AsyncIOSpec {
```

Comment:
Let's use `AnyWordSpec`

Reply:
should be resolved [here](https://github.com/polystat/odin/pull/6/commits/b33dae2b72befc8cd3ee579c011433f5317c43de)